ground/src/main/java/com/google/android/ground/persistence/local/room/stores/RoomSubmissionStore.kt

```diff
@@ -226,13 +226,13 @@ class RoomSubmissionStore @Inject internal constructor() : LocalSubmissionStore
       .map { mutations ->
         mutations.filter { it.surveyId == survey.id }.map { it.toModelObject(survey) }
       }
-      .catch { Timber.e("Ignoring invalid submission mutation", it) }
+      .catch { Timber.e("Ignoring invalid submission mutation $it") }
```

Comment:
We should change the order of throwable to Timber.e(it, "msg")


Timber.e(throwable, "error message", extra msg args)

Reply:
Ah, sorry, I missed that this is actually the exception.